hackathon_site/hardware/test_api.py

```diff
@@ -136,6 +136,29 @@ def test_category_get_success(self):
 
         self.assertEqual(expected_response, data["results"][0])
 
+    def test_hardware_quantity_success(self):
+        self._login()
+
+        category2 = Category.objects.create(name="Microcontrollers", max_per_team=4)
+        hardware1 = Hardware.objects.create(name="Arduino", quantity_available=2)
+        hardware2 = Hardware.objects.create(name="ESP32", quantity_available=3)
+
+        hardware1.categories.add(category2)
+        hardware1.categories.add(self.category)
+        hardware2.categories.add(category2)
+
+        response = self.client.get(self.view)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.json()
+
+        expected_unique_hardware_counts = [1, 2]
+        actual_unique_hardware_counts = [
+            result["unique_hardware_count"] for result in data["results"]
+        ]
+
+        print(data)
```

Comment:
Remove this print statement, otherwise it's going to appear in the logs in the test runner